src/feedback/vue/submitFeedback.vue

```diff
@@ -71,6 +74,7 @@ export default defineComponent({
             isSubmitting: false,
             error: '',
             feedbackName: '',
+            extraCodeTransformData: { jobId: '', clientId: '' },
```

Comment:
why is this needed? it's not used anywhere except below, where it could be a local variable.

this should be a separate PR.

Reply:
Will fix and do in separate PR